### ISSUE TITLE
As a front end user, I want an "apply" button in the filters panel, so that I can filter results without closing the filters panel and clicking the submit button.

### ISSUE THREAD

**Issue by blms** [Princeton-CDH/geniza#646]
## dev notes

Split off as a separate issue so that the filters feature can be tested sooner.

- [ ] requires implementing the deferred part of #450 (button styles)
- [ ] clicking "apply" should submit the form

**Comment by gissoo** [comment]
@blms thank you! these work well. There are three issues that are happening: 1. the results are still being sorted by "random" after I submit the search on all browsers 2. when I tab through the filters and press enter to select them instead the filter window goes away. 3. also the even after refreshing the site the filters aren't resetting, only a hard refresh is working, not sure if I'm doing something wrong

**Comment by blms** [comment]
@gissoo Thanks for testing!

1. Did you enter text into the search box? If so, it should change to "relevance." If not, I think that's the expected behavior.
2. That's the default browser behavior: spacebar toggles checkboxes on and off, and enter submits the form. Should we override that?
3. Can you give more detail about the expected behavior? Right now we don't intend to reset filters when refreshing the page: in fact I think they are meant to stay applied, so that someone could copy the URL and send it to someone else and they would get the same search with all the same filters applied. Should we implement some kind of "reset filters" button?

**Comment by rlskoeser** [comment]
@gissoo agree, reset filters sounds valuable

**Comment by blms** [comment]
1. @rlskoeser thoughts on 1, whether default sort should be random with filters applied? Either way I think it is a separate issue from the apply button.
2. Thanks, I guess we'll leave this issue open until then!
3. @gissoo I also agree.

**Comment by rlskoeser** [comment]
@blms I don't think default sort should change depending on whether filters are applied or not, but I do wonder if random sort being the default will be a little disorienting. Maybe sorting by shelfmark would be a better default...

I do think we need to add a little logic to the stimulus search app to track sort state and revert to previously selected sort option when we disable relevance sort.

**Comment by gissoo** [comment]
@rlskoeser agreed!!! it does feel disorienting as a default sort tbh  ..ok bye for real this time :D
